src/kirby/components/empty-state/empty-state.component.tns.html

```diff
@@ -0,0 +1,10 @@
+<FlexboxLayout flexDirection="column" justifyContent="space-around">
+  <StackLayout class="box" alignSelf="center">
+    <kirby-icon *ngIf="iconName" [name]="iconName" size="lg" outline="true" class="icon"></kirby-icon>
+    <Label *ngIf="title" class="kirby-text-normal-bold title" [text]="title"></Label>
```

Comment:
Use `.h3` class as kirby-text-normal-bold will be deprecated:
```suggestion
    <Label *ngIf="title" class="h3 title" [text]="title"></Label>
```

Reply:
Fixed